Jellyfin.Plugin.NextPVR/Jellyfin.Plugin.NextPVR.csproj

```diff
@@ -1,7 +1,9 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
+    <AssemblyVersion>6.0.0.0</AssemblyVersion>
+    <FileVersion>6.0.0.0</FileVersion>
```

Comment:
```suggestion
```

Version is set in `Directory.Build.props`

Reply:
We fixed the issue where Directory.Build.props wasn't being picked up, and using that file is the preferred method

Reply:
So do you know what the add-on shows today for download 5 or 8?  What do I change to show it jump one major number for the API changes.

Reply:
That's easy, I will want to backport a one liner to 10.8 once this is done, but I can figure out minor numbering changes on the PR thread.

Reply:
We won't be making a new plugin release for 10.8, so that isn't needed.